src/_locales/en/messages.json

```
@@ -80,6 +80,10 @@
     "message": "Permission request failed.",
     "description": "Error shown when a permission request failed. This indicates a technical error and does NOT mean the user declined the permission."
   },
+  "errorQrCodeOverflow": {
+    "message": "Cannot generate QR Code, data size too big.",
```

```suggestion
    "message": "Cannot generate QR code, data size too big.",
```

This is how it is written in this project…